src/molecules/formfields/RadioField/radio_field.module.scss

```diff
@@ -40,7 +40,7 @@ $checked-size: rem-calc(14px);
     width: $checked-size;
     height: $checked-size;
     border-radius: 50%;
-    background-color: color('secondary-2');
+    background-color: color('primary-1');
```

Comment:
Under [the new design for radio buttons](https://www.figma.com/file/iVP8PicuOCAkB7NpswyAWK/Mortar---RCL-color-updates?node-id=39%3A1101&viewport=88%2C41%2C0.5263888835906982) the inputs should be the brand primary color.
![image](https://user-images.githubusercontent.com/6582067/106981023-3d082580-672f-11eb-9675-875292c5887b.png)

Reply:
🔥